### PR TITLE
feat: improve small screen view for feed page

### LINKED ISSUE #1510: Bug: we cant see the sidebars on small screens
### Describe the bug

If we go to small or medium screens, there is no way to see the side bars: 

image

### Steps to reproduce

1. go to feed 
2. make the screen small or medium

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Deadreyo**
## Description

This PR brings back the panels in large screens to the smaller views of the Feed page and improves its responsivity.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

fixes #1510 

### Repository list is now compact.
Before: 
image

After:
image

### XS view:

https://github.com/open-sauced/app/assets/52013393/4ad2fa15-5d40-4b15-bdb2-1d48f0c6cd20

### SM & MD view:

https://github.com/open-sauced/app/assets/52013393/04fca918-e665-4e7d-b89b-6b89391edf92

## After deployment tasks

We can move Repository List filters to be above the highlights for small views, but that's for another issue.

### CODE REVIEW

**Comment by Deadreyo** [comment]
@bdougie @nickytonline @OgDev-01 ready for review

**Review by jpmcb — CHANGES_REQUESTED** [review]
This seems to make the responsiveness much worse (at least on firefox):

Screenshot 2023-10-23 at 9 06 53 AM

Compared to `beta.app.opensauced.pizza` around the same size:

Screenshot 2023-10-23 at 9 06 25 AM

---

We shouldn't accept this unless the responsiveness is the same: we've had a few PRs slip in that have caused the responsiveness to degrade. And ideally, we'd develop some selenium / headless chrome tests to block on any breaking changes.

**Comment by OgDev-01** [comment]
I missed the issue for this.  We've had discussions on improving the UX on mobile in general and the @open-sauced/design are looking into that. Thanks @Deadreyo for turning this around so quickly, but ATM, I don't think this is needed. cc @isabensusan and @nickytonline

**Comment by Deadreyo** [comment]
@jpmcb that's a little side-effect that I didn't notice and is easily fixed if this PR is going to continue.

@OgDev-01 
https://github.com/open-sauced/app/issues/1510#issuecomment-1774101224
So is the final say that this PR is not needed or is it still awaiting a definitive decision?

File: pages/feed/index.tsx
Comment (NsdHSO) [inline pages/feed/index.tsx:320]: maybe you can remove `0.4rem` and tried with `pt-1.5` -> '0.375rem'?

**Comment by bdougie** [comment]
Looking at my feed on iOS and do not see the repositories filter list.

My assumption that I would see it on the deploy preview 

image

context iPhone 15 using safari

**Comment by OgDev-01** [comment]
> So is the final say that this PR is not needed or is it still awaiting a definitive decision?

Unless an objection is made by @open-sauced/engineering and @open-sauced/design, i think this should be revisited when we have more context 👍

**Comment by Deadreyo** [comment]
> Looking at my feed on iOS and do not see the repositories filter list.
> 
> My assumption that I would see it on the deploy preview

The deploy preview is outdated for some reason. Maybe because of this? https://github.com/open-sauced/app/pull/1985#issuecomment-1775090131

File: pages/feed/index.tsx
Comment (Deadreyo) [reply pages/feed/index.tsx:320]: That was already there when I started so I didn't touch it.

**Comment by nickytonline** [comment]
Hi @Deadreyo, just checking in here. I see there are some conflicts to be resolved.

**Comment by Deadreyo** [comment]
> Hi @Deadreyo, just checking in here. I see there are some conflicts to be resolved.

Sunday said that this PR is not needed so 🤷‍♂️

**Comment by nickytonline** [comment]
@Deadreyo yeah, I kissed that comment. I also see in the issue that it's waiting on feed back from design, so let's hold off on this for now.

**Comment by nickytonline** [comment]
Thanks for the pull request @Deadreyo. I'm going to go ahead and close this for now based on the comments that this is not needed at the moment and if we revisit this it requires feedback from @open-sauced/design.

**Pull request closed without merging**
